Cargo.toml

```diff
@@ -2,5 +2,6 @@
 resolver = "2"
 members = [
   "cli",
+  "parse",
```

Comment:
`parse` is not exactly a descriptive name. I appreciate the brevity but I feel like it might be misplaced for a crate name? Especially if we ever want to publish any of these to crate.io.

I'd at the very least include *what* we're parsing in the name? In fact I'd say that's the more important part here, that we're parsing is mostly implementation detail really.

Reply:
The thinking was that this crate would be for any common parsers because people didn't want to have a lot of repositories. Maybe it doesn't matter anymore since this code has rejoined its original repository.